protected-api-access-protected-api/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
+    "@azure/msal-node": "^1.4.0",
+    "@azure/msal-node-extensions": "^1.0.0-alpha.13",
+    "express": "^4.17.2",
+    "https": "^1.0.0",
```

Comment:
This ships with node, probably shouldn't list it here.  Unless I'm mistaken?

Reply:
You are correct

Reply:
🪓